apps/funding-platform/src/index.ts

```diff
@@ -0,0 +1,85 @@
+import { AccessTokenService } from "./access-token";
+import { getWallet } from "./blockchain";
+import { DBInstance } from "./db";
+import * as api from "./entry-server";
+import { checkFreshRequests } from "./queue";
+import { RequestService } from "./request";
+import { utils } from "rpch-common";
+
+const { log } = utils.createLogger(["funding-platform", "index"]);
+const { SECRET_KEY, WALLET_PRIV_KEY } = process.env;
+const port = 3000;
+const MAX_AMOUNT_OF_TOKENS = 100;
+const TIMEOUT = 30;
+let running = false;
+
+const handleRunning = (state: boolean) => {
+  running = state;
+};
+
+export const start = async (ops: {
```

Comment:
here you export start but not use it anywhere since there is no `index.spec.ts`, in any case, I would favor a pattern like the one in the exit-node, since importing this `start` method will also trigger `main` which u don't want